server/driver/wivrn_hmd.cpp

```diff
@@ -222,4 +235,11 @@ void wivrn_hmd::update_visibility_mask(const from_headset::visibility_mask_chang
 	auto m = visibility_mask.lock();
 	m->at(mask.view_index) = mask.data;
 }
+
+void wivrn_hmd::update_presence(bool presence)
+{
+	std::lock_guard lock(mutex);
+	U_LOG_D("Updating hmd presence to %s", presence ? "true" : "false");
```

Comment:
Events should not happen that often, I would increase level to info and rephrase to "user presence" instead of "hmd presence"